meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -133,6 +133,7 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
+    CONFIG_KEYS=y \
```

Comment:
The actual PR which needs this configuration is: https://github.com/opencontainers/runc/pull/488 . Could you please reference it in the commit log?